crates/sargon/src/system/sargon_os/transactions/support/signable_manifest_summary.rs

```diff
@@ -0,0 +1,151 @@
+use crate::prelude::*;
+
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct SignableManifestSummary {
+    pub id: Exactly32Bytes,
+    pub summary: ManifestSummary,
+}
+
+impl SignableManifestSummary {
+    pub fn new(summary: ManifestSummary) -> Self {
+        Self {
+            id: Exactly32Bytes::generate(),
+            summary: summary,
+        }
+    }
+}
+
+impl HasSampleValues for SignableManifestSummary {
+    fn sample() -> Self {
+        Self {
```

Comment:
prefer using `new` (ctor) for samples.

Reply:
I used this so the `id` is stable, `new` will generated a random `id`